components/sections/floating-info-box.js

```diff
@@ -0,0 +1,24 @@
+import { Box } from "@mui/material/"
+import Markdown from "../elements/markdown"
+
+const FloatingInfoBox = ({ data }) => {
+  return (
+    <div className="container pb-4  text-gray-dark">
+      <Box
+        sx={{
+          float: data.alignment === "left" ? "left" : "right",
+          marginRight: data.alignment === "left" ? "1.2rem" : 0,
+          marginLeft: data.alignment === "left" ? 0 : "1.2rem",
+          marginBottom: "1rem",
+          width: "40%",
+          background: "linear-gradient(315deg, transparent 17px, #e5e0e7 0)",
+          padding: "1rem 1.5rem",
+        }}
```

Comment:
would it be possible/make more sense to target and style the links for this box here in its own styles, instead of passing in the background prop to the link?

```
sx={{
  // ...
  a: {
    backgroundColor: '#982568',
  }
}}
```

Reply:
i think that makes sense, but in this case, since they decided not to use the dark background in the end, i've decided to just remove all of the darkBackground props. if a dark background comes up again in the future, we can keep this in mind!